src/mumble/VoiceRecorder.cpp

```diff
@@ -277,6 +289,15 @@ bool VoiceRecorder::ensureFileIsOpenedFor(SF_INFO &soundFileInfo, boost::shared_
 		return false;
 	}
 
+	//Enable constant 320kbps for MP3 formats
+	if (m_config.recordingFormat == VoiceRecorderFormat::MP3)
```

Comment:
This entire ``if`` will throw a compile error, if MP3 is not available as the enum value will not be declared here.